### DISCUSSION TITLE
Springboot JdbcTemplate with Planetscale MySQL

### DISCUSSION THREAD

**Discussion by washmegrp** [planetscale/discussion#444] [Q&A]
Hi All,

I am wondering could we have a connection using jdbctemplate?
I tried to apply the connections but it return me as `Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Access denied for user 'xxxxxxxxxxxxxxxxx'`

My _**pom.xml**_
`<dependency>`
`	<groupId>org.springframework.boot</groupId>`
`	<artifactId>spring-boot-starter-jdbc</artifactId>`
`</dependency>`
`<dependency>`
`	<groupId>mysql</groupId>`
`	<artifactId>mysql-connector-java</artifactId>`
`</dependency>`

My _**application.properties**_
`spring.datasource.url=jdbc:mysql://aws.connect.psdb.cloud/database_name`
`spring.datasource.username=xxxxxxxxxxxxxxxxx`
`spring.datasource.password=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx`
`spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver`

**Comment by fooforge** [comment]
Hey @washmegrp,

We have an example for the JDBC Driver in [planetscale/connection-examples](https://github.com/planetscale/connection-examples/tree/main/java) and have had a short conversation around Spring Boot in https://github.com/planetscale/discussion/discussions/208#discussioncomment-2907613 last year. Mind checking those two resources out and compare these to what you have configured?

I notice the `sslMode` portion is missing from your URL. Though, I would have expected an SSL-centric error message if that poses a problem so please also check if your credentials are actually valid.

Let me know how you get on. Thanks!
